dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1029,7 +1076,10 @@ public void shouldReturnPageHTMLForLegacyUUID() throws Exception {
      * @throws Exception
      */
     @Test
-    public void shouldReturnPageHTMLForLegacyUUIDAndMultiTree() throws Exception {
+    @UseDataProvider("cases")
+    public void shouldReturnPageHTMLForLegacyUUIDAndMultiTree(final Container container, final Template templateTestCase) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)